src/test/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionpostponementrequest/ActionPostponementRequestAboutToSubmitHandlerTest.java

```diff
@@ -92,6 +92,7 @@ public class ActionPostponementRequestAboutToSubmitHandlerTest {
     public void setUp() {
         openMocks(this);
         ReflectionTestUtils.setField(handler, "isScheduleListingEnabled", true);
+        ReflectionTestUtils.setField(handler, "isTypeaheadEnabled", true);
```

Comment:
Does this test class need an extra test to prove that when typeahead is not enabled, the other thing happened?

Reply:
Done